plugins/generator-1.18.2/forge-1.18.2/procedures/entity-freeze.java.ftl

```diff
@@ -0,0 +1,2 @@
+

```

Comment:
This line is unnecessary.

Reply:
I added it because on my tests the code was put in the same line if that line wasn't added. Tho it works without it, it still might confuse users imo